connectors/src/connectors/zendesk/temporal/activities.ts

```diff
@@ -392,13 +392,21 @@ export async function syncZendeskTicketBatchActivity({
     brandId,
   });
 
-  const { tickets, meta } = await fetchZendeskTicketsInBrand({
+  const { tickets, meta } = await fetchSolvedZendeskTicketsInBrand({
```

Comment:
aren't the tickets in the `results` field (the fallback return value of `fetchSolvedZendeskTicketsInBrand` points to that)?

Reply:
Not sure what you mean by fallback value 

Response format is : 

```return response
    ? {
        tickets: response.results || [],
        meta: {
          has_more: !!response.meta?.has_more,
          after_cursor: response.meta?.after_cursor || "",
        },
      }
    : { tickets: [], meta: { has_more: false, after_cursor: "" } };
```

So it's exposing 'tickets' and not results :)

Reply:
oh yeah my bad thought the same pattern was used as for the articles

Reply:
all good the